src/utilities/bcl/templates/EnergyPlusMeasure/tests/test_energyplus_measure.py

```diff
@@ -113,3 +118,12 @@ def test_good_argument_values(self):
         # save the workspace to test output directory
         output_file_path = Path(__file__).parent.absolute() / "output" / "test_output.idf"
         workspace.save(output_file_path, True)
+
+
+# This allows running openstudio CLI on this file (`openstudio test_measure.py`, maybe with extra args)
+if __name__ == "__main__":
+    import sys
+    if len(sys.argv) > 1:
+        pytest.main([__file__] + sys.argv[1:])
+    else:
+        pytest.main()
```

Comment:
This bit is really optional, but I think it's nice to be able to do this, like you could do `openstudio measure_test.rb`